src/CI3Compatible/Database/CI_DB_query_builder.php

```diff
@@ -358,6 +362,27 @@ public function order_by(
         return $this;
     }
 
+    /**
+     * GROUP BY
+     *
+     * @param mixed $by     what field or fields to group_by
+     * @param bool  $escape
+     *
+     * @return CI_DB_query_builder
+     */
+    public function group_by(mixed $by, ?bool $escape = null): self
```

Comment:
We cannot use type `mixed` because it is available as of PHP 8.0. 

```suggestion
    public function group_by($by, ?bool $escape = null): self
```

Reply:
fixed